env.sample

```diff
@@ -1,8 +1,18 @@
-AA_TOKEN=token
 CLIENT_URL="https://api.aleph-alpha.com"
 ARGILLA_API_URL="http://localhost:6900/"
 ARGILLA_API_KEY="argilla.apikey"
```

Comment:
Shouldn't the the ARGILLA_API_KEY also be under the "things to adapt" section?

Reply:
this is the default argilla apikey actually